src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1537,7 +1542,13 @@ public bool IsSimpleAssignmentStatement(SyntaxNode statement)
         public void GetPartsOfAssignmentStatement(
```

Comment:
>GetPartsOfAssignmentStatement [](start = 20, length = 29)

Hum, this API now works with Assignments that aren't part of a statement. The name should probably be adjusted, or maybe another API introduced?

Reply:
I ended up changing this to be clearer.  GetPartsOfAssignmentStatement  only works for the statement form.  For the form where you do want to work either on an expression or the statemnet form there's a new helper.